Gemfile.lock

```diff
@@ -71,73 +71,73 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (2.0.0)
       rails (>= 7.0)
-    activejob (7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activejob (7.0.4)
+      activesupport (= 7.0.4)
       globalid (>= 0.3.6)
-    activemodel (7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activerecord (7.0.2.3)
-      activemodel (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activestorage (7.0.2.3)
-      actionpack (= 7.0.2.3)
-      activejob (= 7.0.2.3)
-      activerecord (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activemodel (7.0.4)
+      activesupport (= 7.0.4)
+    activerecord (7.0.4)
+      activemodel (= 7.0.4)
+      activesupport (= 7.0.4)
+    activestorage (7.0.4)
+      actionpack (= 7.0.4)
+      activejob (= 7.0.4)
+      activerecord (= 7.0.4)
+      activesupport (= 7.0.4)
       marcel (~> 1.0)
       mini_mime (>= 1.1.0)
-    activesupport (7.0.2.3)
+    activesupport (7.0.4)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 1.6, < 2)
       minitest (>= 5.1)
       tzinfo (~> 2.0)
-    acts_as_tenant (0.5.1)
+    acts_as_tenant (0.5.2)
       rails (>= 5.2)
       request_store (>= 1.0.5)
-    addressable (2.8.0)
-      public_suffix (>= 2.0.2, < 5.0)
-    airbrussh (1.4.0)
+    addressable (2.8.1)
+      public_suffix (>= 2.0.2, < 6.0)
+    airbrussh (1.4.1)
       sshkit (>= 1.6.1, != 1.7.0)
     annotate (3.2.0)
       activerecord (>= 3.2, < 8.0)
       rake (>= 10.4, < 14.0)
-    apipie-rails (0.7.0)
+    apipie-rails (0.8.2)
```

Comment:
### [apipie-rails](http://github.com/Apipie/apipie-rails)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.7.0 → 0.8.2

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/apipie-rails/0.7.0/0.8.2)_]





<details>
<summary>Commits</summary>

A change of **47** commits. See the full changes on [the compare page](https://github.com/Apipie/apipie-rails/compare/v0.7.0...v0.8.2).

These are the first 10 commits:
- (7cd8393) [add `ko` locale](https://github.com/Apipie/apipie-rails/commit/7cd8393419de54861bf153c9b4db6640aa767dd5)
- (e3795f6) [fix: enable swagger generator to add referenced schema for a…](https://github.com/Apipie/apipie-rails/commit/e3795f6cf9afe3977d1aa509de927a3201ddb6e9)
- (b57ef08) [Added funcionality to skip on non-validated keys](https://github.com/Apipie/apipie-rails/commit/b57ef08a0ecea05d18a976a06b25a2e4292614fc)
- (748b96a) [Added test to delete the param](https://github.com/Apipie/apipie-rails/commit/748b96a35df6b5ae9b6faccae12d05fc30a67e0d)
- (b8ff011) [Changed readme content to include deletion](https://github.com/Apipie/apipie-rails/commit/b8ff0113f655c1e20c897a66b32614a6fbd293c8)
- (399dfa6) [Fixed error in a test](https://github.com/Apipie/apipie-rails/commit/399dfa635e8f260a6c533763c0717d76def82f82)
- (d2e89bc) [upgrade bootstrap to v2.3.2](https://github.com/Apipie/apipie-rails/commit/d2e89bcaa6f0462516845e82cda314dd38a3586a)
- (cdafeeb) [upgrade jquery to version 1.11.3](https://github.com/Apipie/apipie-rails/commit/cdafeeb578b354f8854d82532fc4573f6b2aeb07)
- (38d46a7) [Add support for scheme definition in Swagger docs](https://github.com/Apipie/apipie-rails/commit/38d46a745994ef917165c2b788a62b21ab96b4e4)
- (cc3bcb8) [Support defining security mechanisms for Swagger (#711)](https://github.com/Apipie/apipie-rails/commit/cc3bcb80a02edf2e483db0e4da59047535538141)


</details>